portfolio/src/main/webapp/index.html

```diff
@@ -38,13 +38,31 @@
       <div class="main">
         <iframe class="frame" src="main.html" width="76%" height="96%" id="sub-page"></iframe>
         <div id="comments"></div>
-        <form action="/comment" method="POST" id="comment-box">
-          <label for="name">Preferred name:</label>
-          <input name="name" class="field" type="text"></input>
-          <label>Comments:</label>
-          <textarea name="text" class="field" rows="3" cols="40"></textarea>
-          <input type="submit" id="submit-btn">
-        </form>
+        <div class="form-popup hidden-elem" id="NameForm">
```

Comment:
Be consistent with your naming for ids and classes. We prefer all lowercase with words separated by dashes like you have below.

Reply:
Sorry that's a typo!